apps/mobile/src/components/wallet-settings/account-name-modal.tsx

```diff
@@ -0,0 +1,30 @@
+import { RefObject } from 'react';
```

Comment:
Design want to call all `modal`s as 'Sheet` so we should change the name of this. See https://github.com/leather-io/issues/issues/268 

I guess this also includes these files